### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by arsalansufi**
This PR ports over two security updates from FamPo:

- [Install and use helmet, FamPo PR 58](https://github.com/Clever/family-portal/pull/58)
- [Add meta referrer origin tag, FamPo PR 44](https://github.com/Clever/family-portal/pull/44)

See those PRs for more context!

### CODE REVIEW

**Comment by arsalansufi** [comment]
For context @sriram-clever, we use this repo as a template when initializing new frontend repos, e.g. `ark init new-frontend-service --template template-frontend`. Let me know if I can provide any further context!

File: src/server/pages/views/common.pug [hunk 1]
```diff
@@ -1,4 +1,7 @@
 mixin commonHeadContent
   title Clever | {{.AppName}}
   link(rel="shortcut icon" href="/favicon.ico?1")
+  //- Only include the origin and not query params in referrer headers to avoid exposing sensitive
+  //- query params to third parties
+  meta(name="referrer" content="origin")
```
Comment (sriram-clever) [inline src/server/pages/views/common.pug:6]: This should probably be `strict-origin` but wondering if `strict-origin-when-cross-origin` would be a better choice. I see that Safari on iOS doesn't support these so might be a reason to not do `strict-*` :(

**Review by sriram-clever — CHANGES_REQUESTED** [review]
Minor question on the choice of the value for referrer. If this choice was related to Safari on iOS support I would suggest at least a comment to revisit this.

File: src/server/pages/views/common.pug (hunk 1 above)
Comment (arsalansufi) [reply src/server/pages/views/common.pug:6]: Ah very interesting, thanks for the context.

The decision to use `origin` in this PR was based on code in existing Clever repos. In addition to family-portal, which I linked to in the PR description, [`oauth` also uses `origin`](https://github.com/Clever/oauth/blob/8aeb1a7dc9e7d37e7e5304adfdd54e24de928da8/legacy-views/layout.jade#L6-L7). I actually didn't know of these other options.

Reading through the different policies, `strict-origin-when-cross-origin` does seem ideal. But the lack of support on Safari on iOS is a dealbreaker for now. Will add a comment!

Comment (arsalansufi) [reply src/server/pages/views/common.pug:10]: https://github.com/Clever/template-frontend/pull/45/commits/71a72adaf9888961516228705322897d82fc4c55

**Review by sriram-clever — APPROVED** [review]
lgtm

**Pull request merged**